openapi/tools/merge_openapi.py

```diff
@@ -0,0 +1,144 @@
+import sys
+import argparse
+import yaml
+from glob import glob
+from openapi_spec_validator import validate_spec
+from openapi_spec_validator.readers import read_from_filename
+from pathlib import Path
+
+global_had_load_error = False
+
+def validate_openapi(filepath):
```

Comment:
most of these top-level methods can be private methods of your class.

Reply:
I moved the class-related ones into the class.